rfcs/text/0030/file.yml

```diff
@@ -0,0 +1,45 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: file
+  reusable:
+    expected:
+      - at: file
+        as: file
+        short_override: Details about a file
+      - at: file_description
```

Comment:
Is `file_description` distinct from the `fd`/`file_descriptor` fields?

Reply:
file_description is the new file_description.yml type. Based on the type of FD, it will have different nested fields. e.g if fd.type is 'pipe' there will be a fd.pipe<File>,   if type is 'fifo', there will be a file_description.fifo<File> with a subset of file props set. etc... Currently the only 4 fd types which have variying sub props, are types: char_device, file, pipe, fifo and socket.

Reply:
I see the issue now, will wait to hear back from Mike about this fieldset name, and will update all the spots that reference it.